mmcv/runner/hooks/optimizer.py

```diff
@@ -99,6 +99,10 @@ def before_run(self, runner):
             """Preparing steps before Mixed Precision Training."""
             # wrap model mode to fp16
             wrap_fp16_model(runner.model)
+            # resume from state dict
+            if 'fp16.loss_scaler' in runner.meta:
+                scaler_state_dict = runner.meta['fp16.loss_scaler']
```

Comment:
I'm not sure whether `fp16.loss_scalar` is suitable here. I may suggest using `_` instead of `.`.

Reply:
I suggest we simply use `['fp16']` in cause there are other related keys of FP16 in the future.